README.rst

```diff
@@ -55,6 +55,17 @@ extendable and language independent. coala is written with a lower case
 
 Read more at our `documentation <http://coala.rtfd.org/>`__.
 
+Why use coala?
+--------------
+
+- coala provides built-in checking routines (named bears in coala).
+- Serves your requirement: You can easily write your own checks (using bears).
+- coala provides user-friendly interfaces like json, formatted and interactive
+  output in the CLI and plugins for various editors are available as well.
+- Optimal Performance: coala manages parallelizing the checking-routines
+  without you having to worry.
+- Unified Interface: One tool to take care of other tools.
```

Comment:
@Makman2  : Will proceed that way then :smile:  :+1:  
@sils1297 : Done the changes ! :smile:  :+1: 
![screenshot from 2016-03-17 00-53-47](https://cloud.githubusercontent.com/assets/8680142/13825904/d7ece03e-ebda-11e5-9ceb-91c33a78535d.png)

good to go now ?